theme/DarkTheme.ts

```diff
@@ -1,10 +1,16 @@
-import { createTheme, responsiveFontSizes } from "@mui/material/styles";
+import { createTheme, adaptV4Theme, responsiveFontSizes, Theme } from "@mui/material/styles";
 
-const oceanic = responsiveFontSizes(createTheme({
+const oceanic: Theme = responsiveFontSizes(createTheme(adaptV4Theme({
     palette: {
-        mode: "dark",
+        type: "dark",
+        common: {
+            white: "#009688",
+        },
         primary: {
             main: "#009688",
+            light: "#009688",
+            dark: "#009688",
+            main: "#009688",
```

Comment:
@vincenzopalazzo What is this `hint` used for under the `text`?

Reply:
I added this to debug because yesterday I spend a lot of time to debug why the theme was not working (a but introduced by a machine damnt!)

There is the docs of the theme https://mui.com/material-ui/customization/default-theme/?expand-path=$.palette.primary where you can find all the details